src/components/Header/styles.module.css

```diff
@@ -1,13 +1,22 @@
 .header {
   border: 4px solid #253f58;
   display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
   align-items: center;
   justify-content: space-between;
   margin: 1rem;
   padding: 2rem 4rem;
   text-align: center;
 }
 
+@media screen and (max-width: 411px) {
```

Comment:
@dcalacci I'm curious. Where does the magic 411px number come from?

Reply:
should actually change all to be 768 (tablet and below). there was a media query at 410, but 411 supports i.e. pixel 2 xl, larger portrait phones.